app/i18n.js

```diff
@@ -0,0 +1,20 @@
+import i18n from 'i18next';
```

Comment:
What about having a `i18n` folder, moving this file as `i18n/index.js` and pushing JSON files from `lang` folder into `i18n/lang` one?

Reply:
Good idea, I have done this.